modello-plugins/modello-plugin-jsonschema/src/main/java/org/codehaus/modello/plugin/jsonschema/JsonSchemaGenerator.java

```diff
@@ -107,7 +107,7 @@ private void generateJsonSchema( Properties parameters )
                                   .enable( JsonWriteFeature.QUOTE_FIELD_NAMES.mappedFeature() )
                                   .enable( JsonWriteFeature.QUOTE_FIELD_NAMES.mappedFeature() )
                                   .disable( JsonWriteFeature.WRITE_NUMBERS_AS_STRINGS.mappedFeature() )
-                                  .createGenerator( schemaFile, JsonEncoding.UTF8 );
+                                  .createGenerator( newWriter( schemaFile.toPath() ) );
```

Comment:
This is wrong. JSON is by definition UTF-8 and nothign else. I think this breaks it.

Reply:
I've raised https://github.com/codehaus-plexus/modello/pull/185